.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-8.0-bookworm",
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {},
+		"ghcr.io/devcontainers/features/powershell:1": {},
+		"ghcr.io/dapr/cli/dapr-cli": {
```

Comment:
Why do we need the dapr-cli? It should only be needed if someone is actually working on the dapr extension/playground app, right?

Reply:
yes I agree. This is for me a big question mark too. On the other hand, should someone who wants to contribute, think about that stuff, or should it be just there?

I mean that's the nice thing about devcontainers that you can just start developing, because someone else took care of installing all the tools you need. 

So I don't know what todo here.
If we install dapr we should maybe install other stuff too. (azure emulators, other CLIs, etc....)

Reply:
I suggest just the basics in this PR and we can add more as requested/needed

Reply:
@danmoseley what are the basics for you? Dapr cli and Azure cli? Just Azure cli? None of both? More?

Reply:
Ha, good point, this repo explicitly isn't cloud specific. @eerhardt any opinion?

Reply:
I've never used the dapr cli and I work in the repo every day.

My opinion would be to not include the dapr cli or the Azure cli initially.

If someone wants these, they can customize their own container right? https://docs.github.com/codespaces/setting-your-user-preferences/personalizing-github-codespaces-for-your-account